### PR TITLE
document process arg for images_upload_handler

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-564

Description of Changes:
* add missing 'progress' argument for `images_upload_handler`
* consolidate duplicate pages about handling images

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/configuration/images-upload-handler.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 The **images_upload_handler** option allows you to specify a function that is used to replace {{site.productname}}'s default JavaScript upload handler function with custom logic.
 
-The upload handler function takes three arguments: `blobInfo`, a `success` callback, and a `failure` callback. When this option is not set, {{site.productname}} utilizes an `XMLHttpRequest` to upload images one at a time to the server and calls the success callback with the location of the remote image.
+The upload handler function takes four arguments: `blobInfo`, a `success` callback, a `failure` callback, and progress value (between 1 and 100). When this option is not set, {{site.productname}} utilizes an `XMLHttpRequest` to upload images one at a time to the server and calls the success callback with the location of the remote image.
```
Comment (lnewson) [inline _includes/configuration/images-upload-handler.md:5]: Should `progress` be surrounded by backticks like the others? Also, should this be included in the example below?

Comment (tylerkelly13) [reply _includes/configuration/images-upload-handler.md]: _May_ have been fixed by: https://github.com/tinymce/tinymce-docs/pull/1565/commits/aeb994c8e1d04b146cff03990b8617ddc4e7e0ee

File: _includes/configuration/images-upload-handler.md [hunk 2]
```diff
@@ -2,9 +2,16 @@
...
 The **images_upload_handler** option allows you to specify a function that is used to replace {{site.productname}}'s default JavaScript upload handler function with custom logic.
 
-The upload handler function takes three arguments: `blobInfo`, a `success` callback, and a `failure` callback. When this option is not set, {{site.productname}} utilizes an `XMLHttpRequest` to upload images one at a time to the server and calls the success callback with the location of the remote image.
+The upload handler function takes four arguments:
 
->Note: Please note that when using this option, no other image uploader options are necessary. Additionally, if you would like {{site.productname}} to replace the `<image>` tag's `src` attribute with the remote location, please use the success callback defined in the `images_upload_handler` function with the returned JSON object's location property.
+* `blobInfo`
+* A `success` callback
+* A `failure` callback
+* A `progress` callback (returning a value between 1 and 100)
```
Comment (lnewson) [inline _includes/configuration/images-upload-handler.md:10]: The returning part here is incorrect. It takes a value between 1 and 100 when called, but doesn't return anything.

File: _includes/configuration/images-upload-handler.md [hunk 3]
```diff
@@ -13,17 +20,21 @@ The upload handler function takes three arguments: `blobInfo`, a `success` callb
...
   selector: 'textarea',  // change this value according to your HTML
-  images_upload_handler: function (blobInfo, success, failure) {
+  images_upload_handler: function (blobInfo, success, failure, progress) {
     var xhr, formData;
 
     xhr = new XMLHttpRequest();
     xhr.withCredentials = false;
     xhr.open('POST', 'postAcceptor.php');
 
+    xhr.upload.onprogress = function(e) {
```
Comment (lnewson) [inline _includes/configuration/images-upload-handler.md:30]: For consistency, maybe add a space after `function`?

Comment (tylerkelly13) [reply _includes/configuration/images-upload-handler.md]: fixed by: https://github.com/tinymce/tinymce-docs/pull/1565/commits/2d0745a16dbd755d3bdf4cb745c053a8d06110b5

File: _includes/configuration/images-upload-handler.md (hunk 2 above)
Comment (tylerkelly13) [reply _includes/configuration/images-upload-handler.md]: Fixed by: https://github.com/tinymce/tinymce-docs/pull/1565/commits/2d0745a16dbd755d3bdf4cb745c053a8d06110b5

**Pull request merged**